CONTRIBUTING.md

```diff
@@ -237,6 +237,12 @@ $ git commit --amend
 $ git push --force-with-lease origin my-branch
 ```
 
+When the commits in your Pull Request get landed, they will be squashed into
+one commit, with metadata added to the commit message (including links to the
```

Comment:
The commits won't necessarily be squashed into one commit per PR, they'll be squashed into one commit per logical change (it's hard to specify what a logical change is).